requirements.yml

```diff
@@ -2,7 +2,7 @@
 collections:
   - name: https://opendev.org/openstack/ansible-collection-kolla
     type: git
-    version: unmaintained/2023.1
+    version: 2023.1-eom
```

Comment:
I don't think we need this now.

Reply:
Yeah, actually we don't need this while the unmaintained branch is still there. I'll close this one off.